third_party/cargo/remote/BUILD.autocfg-1.0.1.bazel

```diff
@@ -10,9 +10,10 @@ load("@bazel_skylib//lib:selects.bzl", "selects")
 
 # buildifier: disable=load
 load(
-    "@rules_rust//rust:rust.bzl",
+    "@rules_rust//rust:defs.bzl",
```

Comment:
It looks like the newer version of `cargo raze` uses `defs.bzl` instead of `rust.bzl`. In any case, if these changes are needed, can we split it off to a different PR, please?

Reply:
@bgogul 

How is it possible to split the changes caused by `cargo raze` into a separate PR given that the dependency is needed for these code changes to run? Do you mean there should be a PR that adds the needed dependencies first and then a second PR comes after that one to use the new dependencies?